readconf.c

```diff
@@ -508,6 +508,22 @@ default_ssh_port(void)
 	return port;
 }
 
+/*
+ * Gets the value of the TERM environment variable as set in the options,
+ * defaulting to using that of the calling environment.
+ */
+const char *
+get_term(const Options *options) {
+    const char *term = NULL;
+    for (int idx = 0; idx < options->num_setenv; idx++) {
+        if (!strncmp("TERM", options->setenv[idx], 4)) {
```

Comment:
Should this be `strncmp("TERM=", options->setenv[idx], 5)` ?

Reply:
Yes, that's indeed buggy on environment variables starting with TERM that aren't TERM. Good eye!

Reply:
I've fixed this and force-pushed.